test/sensor_simulator/gps.cpp

```diff
@@ -15,7 +15,17 @@ Gps::~Gps()
 
 void Gps::send(uint64_t time)
 {
+	const float dt = float((time - _gps_data.time_usec) * 1e-6);
```

Comment:
```suggestion
	const float dt = static_cast<float>((time - _gps_data.time_usec) * 1e-6);
```

Reply:
What is your view on starting to use static_cast<>?

Reply:
@kamilritz I'm for it because it's safer.